build.gradle

```diff
@@ -18,13 +18,11 @@ buildscript {
     repositories {
         maven { url "https://plugins.gradle.org/m2/" }
     }
-    dependencies {
-        classpath 'gradle.plugin.org.jetbrains:gradle-intellij-plugin:0.0.43'
-    }
 }
 
 plugins {
     id 'net.researchgate.release' version '2.3.4'
+    id 'org.jetbrains.intellij' version '0.1.5'
```

Comment:
There was something weird going on, maybe it had something to do with subprojects? I really have no idea... it was so long ago, check the logs?!